s4e-web/src/app/views/map-view/state/map/map.service.ts

```diff
@@ -14,4 +14,8 @@ export class MapService {
               private CONFIG: S4eConfig
   ) {
   }
+
+  toggleZKOptions(open: boolean = true) {
```

Comment:
Shouldn't it be called `showZKOptions` or `setZKOptionsOpened`? Toggle may be misleading